public/app/percona/add-instance/components/AddRemoteInstance/FormParts/NodesAgents/NodesAgents.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { render, screen, waitFor } from '@testing-library/react';
```

Comment:
I would expect more tests here regarding the nodes/agents selection:
1. In case of multiple agents on a node - pmm-server is selected
2. In case of a node change - change list of agents (that isnt what the test is testing for)
3. Agent field selects only show `pmm-agent` type agents

etc...